### DISCUSSION TITLE
How to use golang-migrate cli with sqlite

### DISCUSSION THREAD

**Discussion by davidshare** [golang-migrate/migrate#791] [Q&A]
I am working on a desktop application with wails and using SQLite as the preferred database.

I have installed the CLI tool but I can't seem to find any documentation on how to use the migrate command to create migration in sqlite.

I created a makefile to do that for me. I am able to run the command for Postgres databases, but it is not working for SQLite. 

Apart from the fact that I am not sure about the correct syntax, when I run the current script I have, it doesn't return any error, but the database is unaffected.

```
migrateup:
	migrate -path ./backend/database/sqlite/data/ -database $(DB_URL) -verbose up
```

I am using the above command in my makefile the `$(DB_URL)` variable is the filename of the SQLite file, and it is gotten from the .env file.

Please how can I achieve this?

**Comment by dhui** [comment]
https://github.com/golang-migrate/migrate/tree/master/cmd/migrate#usage

**Comment by dhui** [comment]
Closing due to age and lack of activity
